languages/rust/oso/tests/test_oso.rs

```diff
@@ -157,3 +158,29 @@ fn test_allow_model() -> oso::Result<()> {
 
     Ok(())
 }
+
+#[test]
+fn test_get_allowed_actions() -> oso::Result<()> {
+    common::setup();
+    let oso = test_oso();
+
+    let actor = Actor::new(String::from("sally"));
+    let resource = Widget::new(1);
+    let actions: HashSet<Action> = oso.get_allowed_actions(actor, resource)?;
+
+    assert!(actions.len() == 3);
+    assert!(actions.contains(&Action::Typed("CREATE".to_string())));
+    assert!(actions.contains(&Action::Typed("READ".to_string())));
+    assert!(actions.contains(&Action::Typed("get".to_string())));
+
+    let actor = Actor::new(String::from("sally"));
+    let resource = Widget::new(1);
+    let actions: HashSet<String> = oso.get_allowed_actions(actor, resource)?;
+
+    assert!(actions.len() == 3);
+    assert!(actions.contains(&"CREATE".to_string()));
+    assert!(actions.contains(&"READ".to_string()));
+    assert!(actions.contains(&"get".to_string()));
```

Comment:
is the `to_string` necessary? That would be way nicer if not

Reply:
Fixed.